src/dotnet/Chat.UI.Blazor/Components/ChatHeader.razor

```diff
@@ -43,15 +57,44 @@
     [Inject] private ModalUI ModalUI { get; init; } = null!;
     [Inject] private RightPanelUI RightPanelUI { get; init; } = null!;
     [Inject] private NavbarUI NavbarUI { get; init; } = null!;
+    [Inject] private UICommander UICommander { get; init; } = null!;
+    [Inject] private Session Session { get; init; } = null!;
```

Comment:
@FrolYo let's use "standard" order of injections - Session, replica/compute services, UI services, etc.

Reply:
done